packages/shell-api/src/shell-internal-state.ts

```diff
@@ -308,7 +308,7 @@ export default class ShellInternalState {
           const collectionNames = await this.currentDb._getCollectionNamesForCompletion();
           return collectionNames.filter((name) => name.startsWith(collName));
         } catch (err) {
-          if (err.code === ShellApiErrors.NotConnected) {
+          if (err.code === ShellApiErrors.NotConnected || err.codeName === 'Unauthorized') {
```

Comment:
Should `'Unauthorized'` be also part `ShellApiErrors` to be consistent with the other?

Reply:
This is not a shell error, though, it’s a server error :)

Reply:
Makes sense!